core/templates/dev/head/pages/exploration_editor/simple_editor_tab/StatesToQuestionsService.js

```diff
@@ -160,6 +160,15 @@ oppia.factory('StatesToQuestionsService', [
         // EndExploration.
         var state = SimpleEditorShimService.getState(missingStateNames[0]);
         var interactionId = state.interaction.id;
+        SimpleEditorShimService.saveInteractionId(
+              currentStateName, DEFAULT_INTERACTION.ID);
```

Comment:
Also I am a bit confused about why this code is here. If you read the function, it does a lot of verification. Why are we saving stuff?